docs/topics/settings.rst

```diff
@@ -275,6 +275,15 @@ project name). This name will be used for the logging too.
 It's automatically populated with your project name when you create your
 project with the :command:`startproject` command.
 
+.. setting:: CHECK_SETTINGS_ENABLED
+
+CHECK_SETTINGS_ENABLED
+----------------------
+
+Default: ``True``
+
+Whether to enable Check Settings.
```

Comment:
We should either link back to the CheckSettings extension documentation here, or remove this altogether and add `.. setting:: CHECK_SETTINGS_ENABLED`` before the CheckSettings extension documentation header instead.

Reply:
I added `Scope` to settings and a new section for `CHECK_SETTINGS_IGNORED`.